lstchain/io/tests/test_config.py

```diff
@@ -1,6 +1,7 @@
+import tempfile
 from lstchain.io import config
 from lstchain.io.config import get_cleaning_parameters
-import tempfile
+from lstchain.scripts.lstchain_dl1ab import includes_image_modification
```

Comment:
shouldn't be from `lstchain.io.config`. How does this even work?

Reply:
Why shouldn't it?

Reply:
I was expecting that you could not import a function defined somewhere else. But, true, it is not the case

Reply:
@morcuended thank you Daniel, you are right, it should correctly be imported from config, I fixed this.
It worked only because the function was imported by `lstchain_dl1ab` in the first place.